lib/collector/RexModuleInputValueCollector.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace rexstan;
+
+use PhpParser\Node;
+use PHPStan\Analyser\Scope;
+use PHPStan\Collectors\Collector;
+use PHPStan\Node\FileNode;
+
+final class RexModuleInputValueCollector implements Collector {
+    const MODULE_INPUT_FILE_SUFFIX = '.input.php';
+
+    public function getNodeType(): string
+    {
+        return FileNode::class;
+    }
+
+    public function processNode(Node $node, Scope $scope)
+    {
+        if (strpos($scope->getFile(), self::MODULE_INPUT_FILE_SUFFIX) === false) {
+            return null;
+        }
+
+        $x;
```

Comment:
hier haben wir den pfad zur datei im filesystem zum module-input.

jetzt bräuchte man einen weg wie man die datei durch den rex-var mechnismus laufen lassen kann bei dem man die gefunden rex-vars zurückbekommt. bisher kann `rex_var` nur direkt mit den gefunden vars ersetzungen durchführen.

@gharlan ich denke es wäre es sinnvoll wenn [das parsen und ersetzen](https://github.com/redaxo/redaxo/blob/main/redaxo/src/core/lib/var/var.php) in 2 schritte aufteilen könnten, sodass man auch auf dem parsing result arbeiten kann, wie es hier für den use-case nötig wäre (siehe PR beschreibung).

du bist im parsen gut im thema.. könntest du dir das vorstellen/bauen?

Reply:
ein erster versuch https://github.com/redaxo/redaxo/pull/5404